### PR TITLE
Fix #175 and #174, Add CI for spell and kickstart check

### LINKED ISSUE #175: Use CI to do spell check for files under docs and readme for each pull request
https://github.com/projectatomic/atomic-site  uses similar CI

### PR DESCRIPTION
**Pull request by praveenkumar**
This will have fix for spellcheck and kickstart checks for each commit. Currently `adb.dict` contains all words which we need to ignore for spellcheck. It is added to travis CI.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "2.7"
+before_install:
+  - apt-get install -y python-pykickstart
+  - apt-get install -y python-enchant
+script:
+  - "ksvalidator build_tools/kickstarts/centos-7-kubernetes-vagrant.ks"
```
Comment (LalatenduMohanty) [inline .travis.yml]: The ks file name is centos-7-adb-vagrant.ks

**Comment by praveenkumar** [comment]
@tkdchen install pyenchant have issue with pip http://stackoverflow.com/questions/29381919/importerror-the-enchant-c-library-was-not-found-please-install-it-via-your-o so I had to go with system package way.

**Comment by LalatenduMohanty** [comment]
@praveenkumar is the job working fine? code LGTM

**Comment by praveenkumar** [comment]
Yes you can check success build -> https://travis-ci.org/projectatomic/adb-atomic-developer-bundle/builds/101533956

**Comment by bexelbie** [comment]
LGTM, once squashed to a single commit.

**Comment by praveenkumar** [comment]
@LalatenduMohanty @bexelbie Done

**Comment by LalatenduMohanty** [comment]
merging

**Pull request merged**